src/main/kotlin/com/cognifide/gradle/aem/instance/content/ContentManipulator.kt

```diff
@@ -0,0 +1,16 @@
+package com.cognifide.gradle.aem.instance.content
+
+interface ContentManipulator {
+
+    fun getNode(path: String): Node?
+
+    fun createNode(path: String, props: Map<String, Any> = mapOf()): Node
```

Comment:
I think that such `mapOf()` as default value will be error prone;

please remove it; such value should be intentionally specified; same for below methods